README.md

```diff
@@ -43,24 +43,35 @@ snowdrop	    https://snowdrop.github.io/helm
 To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
 
 ```console
-$ helm create rest-http
+$ mkdir rest-http
 ```
-This command will generate the following directory structure using th Helm starter template:
+
+Now, let's create the `Chart.yaml` file at `rest-http/` with your Chart information:
```

Comment:
... `file at `rest-http/` ... ==> ... under the `rest-http/` folder

Reply:
changed